README.md

```diff
@@ -183,9 +195,11 @@ validation report. There is also a `check()` method that returns an Exception
 only with the first error found:
 
 ```php
+import Respect\Validation\Exceptions\ValidatorExceptionInterface
+
 try {
     $usernameValidator->check('really messed up screen#name');
-} catch(\InvalidArgumentException $e) {
+} catch(ValidatorExceptionInterface $e) {
     echo $e->getMainMessage();
```

Comment:
`check()` may throws `ValidatorException`, like `NotEmptyException`, `NoWhitespaceException` which is not type of `ValidatorExceptionInterface`.